### DISCUSSION TITLE
error's

### DISCUSSION THREAD

**Discussion by billyboi2743** [3xploitGuy/lazyaircrack#6] [Q&A] [answered]
i get this error: [Selected] Option 1 Wifi Hacking... nl80211 not found. socket(PF_PACKET) failed: Operation not permitted This program requires root privileges. [wlan0mon] Preparing for scan... ./lazyaircrack.sh: line 141: kill: (2664) - No such process sed: can't read generated-01.kismet.csv: No such file or directory SerialNo        WiFi Network cut: generated-01.kismet.csv: No such file or directory wc: generated-01.kismet.csv: No such file or directory ./lazyaircrack.sh: line 147: [: 1000: unary operator expected ./lazyaircrack.sh: line 151: generated-01.kismet.csv: No such file or directory ./lazyaircrack.sh: line 152: generated-01.kismet.csv: No such file or directory ./lazyaircrack.sh: line 153: generated-01.kismet.csv: No such file or directory [] Preparing for attack... [wlan0mon] Sending DeAuth to target... ./lazyaircrack.sh: line 71: x-terminal-emulator: command not found [Input] Path to wordlist (Press enter to use default): [Status] Waiting for Handshake Packet... [Status] Checking for Handshake Packet... [!] Can't find Handshake, waiting ... [Status] Checking for Handshake Packet...

do you know why?

**Comment by 3xploitGuy** [comment, marked as answer]
Hi @billyboi2743, you need to run the script as root user. Do **sudo -i** it will ask you for root password enter it and then run the script.

**Comment by billyboi2743** [comment]
ight thx

**Comment by 3lectrowizard** [comment]
anyone know what o do about this???



us] Checking for Handshake Packet...
[!] Can't find Handshake, waiting ...
[Status] Checking for Handshake Packet...
./lazyaircrack.sh: line 86: kill: (2225) - No such process
rm: cannot remove 'handshake-01.cap': No such file or directory
[!] Unknown error is occured...
[GitHub] You can discuss at https://github.com/3xploitGuy/lazyaircrack/discussions
[GitHub] Do paste content of password and error log files...
                                                                                                                                                                     
┌──(root💀lenovo)-[/home/bender/lazyaircrack]

**Reply by 3xploitGuy** [reply]
@3lectrowizard The next version of script is released today, these issues are handled in a much more better way. You can try deleting current version and cloning new, if you face any issue definitely we can discuss.

**Comment by 3lectrowizard** [comment]
Thank you for your response

**Comment by blackdyer26** [comment]
Hey Friend , I found this error on my VM .
![WhatsApp Image 2024-12-26 at 13 31 24_00f10bc7](https://github.com/user-attachments/assets/c89ea67d-2768-4e55-a0e7-23985088e217)
